raster/r.out.vtk/r.out.vtk.html

```diff
@@ -97,11 +97,11 @@ <h3>Spearfish example with RGB data</h3>
 g.region n=4926990 s=4914840 w=591570 e=607800 res=30 -p
```

Comment:
The computational region should be changed to follow the region and coordinate system of the WMS used. The same applies to the title -- it is not `Spearfish example with RGB data` anymore.